tssc/__init__.py

```diff
@@ -1,5 +1,25 @@
-"""
-Trusted Software Supply Chain Library (tssc) main entry point.
+"""Trusted Software Supply Chain Library (tssc) main entry point.
+
+Command-Line Options
```

Comment:
@itewk The help output incorrectly lists required fields as optional. There is a bug already created for this (https://projects.engineering.redhat.com/browse/NAPSSPO-523), but I wanted to bring to your attention while you're working in this area in case it just required a minor tweak. Ignoring this won't prevent acceptance of this PR.

Reply:
@HunterGerlach i noticed that when i copy pasted the help message. I double checked the code and the params are marked as required and the library fails if you don't supply them. It seems to be some bug with ArgumentParser. Not saying we shoulnd't fix. Just maybe trickier / require some kind of merge request to the ArgumentParse project, or maybe we are missing some flag on how we print the help. 

I think this doc is accurate as it is, though it doesn't call out required vs optional. Thanks for opening the other jira.